### PR TITLE
adds "How to create component" page

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Copying across https://github.com/BBC-News/simorgh/wiki/How-to-create-a-new-component

After merge, let's delete the Wiki page.

### CODE REVIEW

File: docs/Creating-Components.md
Comment (sareh) [inline docs/Creating-Components.md]: Updating the link to be a valid one would be great http://localhost:7080/news/articles/c0000000001o

File: docs/Creating-Components.md
Comment (sareh) [inline docs/Creating-Components.md]: It would be great to clarify that integrating into the Article component should be done separately to the creation of the component and container. This is because we should conduct a full accessibility review / accessibility swarm as part of the integration ticket.

**Review by sareh — COMMENTED** [review]
This looks great. Thanks for adding this documentation, Chris! I've just added a clarifying comment.

File: docs/Creating-Components.md
Comment (ChrisBAshton) [reply docs/Creating-Components.md]: Ok! I didn't know it should 👍 will work on it now. 

So for my understanding - 3 PRs:

1) presentational component
2) container component (and corresponding fixture data and schema updates)
 - does the accessibility swarm happen at this point? Does it then get merged?
3) integrate into Article

Is that right? Or does more happen in PR number 3?

Comment (sareh) [reply docs/Creating-Components.md]: Sorry, I should have explained more clearly. 
For PRs 1 & 2, we do some accessibility testing when the PR gets to the UX/Accessibility review column. 
We use this as a reference to guide the accessibility review step https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology

For PR 3, which is integrating into the Article, when the PR gets to the UX/Accessibility review column, we do a full accessibility swarm, with people from multiple disciplines - UX designers, devs, testers, BAs, etc. 

For this, we have a spreadsheet template that is currently in Google Drive. I will get a blank version to reference here in our documentation.

File: docs/Creating-Components.md
Comment (pjlee11) [inline docs/Creating-Components.md]:
```suggestion
data/schema.yaml describes the Article API definition for web. We need to make it aware of our new component.
```

Currently apps are not using this schema

File: docs/Creating-Components.md
Comment (pjlee11) [inline docs/Creating-Components.md]:
```suggestion
Your component is more than likely a new 'block' in the data feed, so you'll need to add it to the array of which blocks the application should validate:
```

File: docs/Creating-Components.md
Comment (pjlee11) [inline docs/Creating-Components.md]:
```suggestion

```

Containers should not have stories

Comment (pjlee11) [reply docs/Creating-Components.md]:
```suggestion
* index.test.jsx - creates "snapshots" of the component with the various different rendered outputs for the business logic in the container
```
Unless this was meant to be `index.test.jsx`

**Comment by pjlee11** [comment]
Good work Chris I've added a few comments/suggestions. I also think we should mention the Cypress tests at the integration stage for the wiki. This is because we would want to ensure any new component that requires integration testing should be covered in the E2Es before being merged. It is also a good opportunity to document the test teams ownership of the Cypress tests.

File: docs/Creating-Components.md
Comment (pjlee11) [inline docs/Creating-Components.md]: Feels like this and line 108 (`### Add a container`) are the same thing. Maybe this heading and summary is in the wrong place?

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

File: docs/Creating-Components.md
Comment (dr3) [inline docs/Creating-Components.md]:
```suggestion
By now, you should have all the automatic PR checks passing, 2 manual code reviews approved, and all the UX/accessibility/test QA completed.
```

**Review by dr3 — DISMISSED** [review]
👍 LGTM apart from that capitalisation

**Comment by ChrisBAshton** [comment]
Thanks @pjlee11 @drewmcmillan - please give it one more thumbs up (I don't _really_ dismiss your "stale reviews" 😂 )

File: docs/Creating-Components.md
Comment (dr3) [inline docs/Creating-Components.md:146]: Soz one small thing again 🙈 Talking to @sareh she wasn't aware moving away from travis is a thing, is it?

Comment (dr3) [reply docs/Creating-Components.md:146]: Discussion on slack with @sareh agreed we should leave this and change when we refactor the jenkins pipeline

**Review by dr3 — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Pull request merged**
